plugins/repoquery.py

```diff
@@ -18,8 +18,8 @@
 
 from __future__ import absolute_import
 from __future__ import unicode_literals
-from datetime import datetime
 from dnfpluginscore import _
+from dnf.cli.output import PackageFormatter, QUERY_TAGS
```

Comment:
is dnf.cli.output public api ?
It is not good pratice to use undocumented, private api in plugins, I will end in every things is API like in yum
It will make it very hard to change things in the future without break stuff.
I know core plugins is maintained by dnf developers, but people making commuty plugins will look at core plugins for how to make plugins.